packages/frontend-core/src/components/grid/stores/datasources/nonPlus.ts

```diff
@@ -97,34 +90,5 @@ export const initialise = (context: StoreContext) => {
       column: get(initialSortColumn),
       order: get(initialSortOrder) || SortOrder.ASCENDING,
     })
-
-    // Update fetch when filter changes
-    unsubscribers.push(
-      allFilters.subscribe($allFilters => {
-        // Ensure we're updating the correct fetch
-        const $fetch = get(fetch)
-        if (!isSameDatasource($fetch?.options?.datasource, $datasource)) {
-          return
-        }
-        $fetch.update({
-          filter: $allFilters,
-        })
-      })
-    )
-
-    // Update fetch when sorting changes
-    unsubscribers.push(
-      sort.subscribe($sort => {
-        // Ensure we're updating the correct fetch
-        const $fetch = get(fetch)
-        if (!isSameDatasource($fetch?.options?.datasource, $datasource)) {
-          return
-        }
-        $fetch.update({
-          sortOrder: $sort.order || SortOrder.ASCENDING,
-          sortColumn: $sort.column,
-        })
-      })
-    )
```

Comment:
These are needed for inline filtering and sorting to work - what was the reason for removing them?

Normal usage of these features:
- Right click column header and change sorting
- Seaching via inline filtering (hovering over the column header and clicking the search icon, then entering a value)

Reply:
I totally missed that. I actually wanted to ask you, as I did not see the actual usage of that, and I thought it was some legacy code. Fixed!